src/contracts/fund/participation/Participation.ts

```diff
@@ -64,7 +65,7 @@ export class Participation extends Contract {
       investmentAsset: result.investmentAsset,
       investmentAmount: toBigNumber(result.investmentAmount),
       requestedShares: toBigNumber(result.requestedShares),
-      timestamp: new Date(parseInt(`${result.timestamp}`, 10) * 1000),
+      timestamp: toDate(`${result.timestamp}`),
```

Comment:
Good idea with the toDate() helper. But please make it accept accept `any` and then handle the different cases in the function instead of stringifying the value first. That's a bit more readable then.